### ISSUE TITLE
Help with SPI and I2C connected PN532 reader

### ISSUE THREAD

**Issue by ronie19** [pokusew/nfc-pcsc#101]
Is this compatible with PN532 reader connected by SPI/I2C ? 
Or only USB devices ?

**Comment by PTS93** [comment]
You would have to implement the PCSC API which this package is using to talk to NFC readers. This is unlikely to be worth it, you will wanna directly talk to your reader.

**Comment by ronie19** [comment]
> You would have to implement the PCSC API which this package is using to talk to NFC readers. This is unlikely to be worth it, you will wanna directly talk to your reader.

Any possible ways of doing that ? I am trying to get it recognised by pcsc_scan command in terminal. But no way I am able to get it via spi. I am only able to operate it with libnfc.

**Comment by PTS93** [comment]
To be fair, if you are unsure on how to approach that the task of implementing a pcsc driver is probably too much.
libnfc already allows you to interact with the NFC reader and access all its functionality. I would suggest you use that to integrate with your application.

**Comment by jonasclaes** [comment]
ifdnfc is your solution

**Comment by ronie19** [comment]
> ifdnfc is your solution

Yes, I did it. However it worked with 64-bit linux due to buffer length issues in 32-bit linux
